### PR TITLE
(Gh103) Fix erroneous quotes in CCR variable.

### LINKED ISSUE #103: Extra quotes around CCR value in Start-C4bCcmSetup.ps1
1. Extra quotes in the variable are throwing off the source in later calls.
2. No `install` directive for the dotnetcore-windowshosting package

### PR DESCRIPTION
**Pull request by steviecoaster**
This removes the double quotes from the variable, since we are calling it in an array now, which was
causing an extra set of quotes to be around the value, tripping up choco.

Fixes #103.

### CODE REVIEW

**Comment by steviecoaster** [comment]
This was tested _live_ with a customer, prior to raising the PR so it's known-good code.

**Comment by adilio** [comment]
The last commit where you're adding install should be a separate issue and PR. Otherwise, initial commit looks good.

File: Start-C4bCcmSetup.ps1 [hunk 1]
```diff
@@ -92,7 +92,7 @@ $chocoArgs = @('install', 'IIS-ApplicationInit', "--source='windowsfeatures'" ,'
 $chocoArgs = @('install', 'aspnetcore-runtimepackagestore', "--version='3.1.16'", "--source='$Ccr'", '--no-progress', '-y')
 & choco @chocoArgs
 
-$chocoArgs = @('dotnetcore-windowshosting', "--version='3.1.16'", "--source='$Ccr'", '--no-progress', '-y')
+$chocoArgs = @('install', 'dotnetcore-windowshosting', "--version='3.1.16'", "--source='$Ccr'", '--no-progress', '-y')
```
Comment (ryanrichter94) [inline Start-C4bCcmSetup.ps1:95]:
```suggestion
$chocoArgs = @('install', 'dotnetcore-windowshosting', "--version='3.1.16'", "'--source=$Ccr'", '--no-progress', '-y')
```

File: Start-C4bCcmSetup.ps1 [hunk 2]
```diff
@@ -26,7 +26,7 @@ Start-Transcript -Path "$env:SystemDrive\choco-setup\logs\Start-C4bCcmSetup-$(Ge
 
 # DB Setup
 $PkgSrc = "$env:SystemDrive\choco-setup\packages"
-$Ccr = "'https://community.chocolatey.org/api/v2/'"
+$Ccr = 'https://community.chocolatey.org/api/v2/'
 
 $chocoArgs = @('upgrade', 'sql-server-express', 'sql-server-management-studio', '-y', "--source='$Ccr'", '--no-progress')
```
Comment (ryanrichter94) [inline Start-C4bCcmSetup.ps1:31]:
```suggestion
$chocoArgs = @('upgrade', 'sql-server-express', 'sql-server-management-studio', '-y', "'--source=$Ccr'", '--no-progress')
```

**Review by ryanrichter94 — COMMENTED** [review]
Tested this change while on a customer support call. If left alone the source resolves to `''https:community.chocolatey.org/api/v2/''` and fails.

**Comment by steviecoaster** [comment]
Spoke with Rob around this. Merging it as all items in this PR were tested with customer and work.

**Pull request merged**